README.md

```diff
@@ -1,20 +1,35 @@
-# Key Rotation Bot
-
-## Introduction
+<h1 align="center">
```

Comment:
HTML in een markdown file?

Reply:
Ja, zag dat de Key Vault Acmebot dit ook gebruikte om de tekst te centreren. Staat wel nice, niet per se noodzakelijk ofzo :)